setup.cfg

```
@@ -1,9 +1,46 @@
+[metadata]
+name = pysatModels,
+version = file: pysatModels/version.txt
+url = github.com/pysat/pysatModels,
+author = Angeline G. Burrell
+author_email = angeline.burrell@nrl.navy.mil
+description = 'Model-data comparisons for the pysat ecosystem',
+long_description = file: README.md
+long_description_content_type = text/markdown
+classifiers =
+    Development Status :: 4 - Alpha
+    Topic :: Scientific/Engineering :: Physics
+    Intended Audience :: Science/Research
+    License :: BSD
+    Natural Language :: English
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Operating System :: MacOS :: MacOS X
+    Operating System :: POSIX :: Linux
+license_file = LICENSE
+
+[options]
+python_requires = >= 3.5
+setup_requires = setuptools >= 38.6; pip >= 10
+packages = find:
+include_package_data = True
+zip_safe = False
+install_requires = numpy
+		   pandas
+		   pyFoorcastTools
+		   pysat
+		   scipy
+		   xarray
+
 [coverage:run]
 
 [coverage:report]
 
 [flake8]
 max-line-length = 80
+ignore = W503
 
```

```suggestion
ignore = W503
exclude = docs/conf.py
```
Not sure if the exclude statement is strictly needed, as flake8 does not seem to grab the docs by default.

---

Oh, it does, but I also think it's ok to flake8 the docs.  So I would rather not exclude them.

---

That's fine.  I think we had this originally due to some conflicts with the conf.py, but if it passes I'm happy with linting this as well.